hicommonwealth/commonwealth
# 6763 stake growl

## Link to Issue
Closes: #6763 

## Description of Changes
- Modified the Valentine Growl to the Stake Growl
- Modified the image, css, and links 

## "How We Fixed It"

- Changed all the names to `StakeGrowl` 
- Modified all the buttons to go to their designated locations
- Updated all fonts, texts, and image.

## Test Plan
- Make sure you can interact with the growl
- Ensure the button takes you to `/createCommunity`
- Ensure the checkbox removes the growl permanently 
- Ensure the 'Learn More' button takes you to the blog(Which isn't yet posted)
